src/pages/courses/[course].tsx

```diff
@@ -54,7 +50,7 @@ export const getStaticPaths: GetStaticPaths = async () => {
 
   return {
     paths,
-    fallback: false,
+    fallback: 'blocking',
```

Comment:
This basically treats the page as SSR if the route is not found in the pre-generated paths.